x-pack/test/reporting_api_integration/reporting_and_security/validation.ts

```diff
@@ -43,7 +43,7 @@ export default function ({ getService }: FtrProviderContext) {
           { jobParams: createPdfV2Params(1541.5999755859375) }
         );
 
-        await retry.tryForTime(48000, async () => {
+        await retry.tryForTime(60000, async () => {
```

Comment:
Is this to resolve https://github.com/elastic/kibana/issues/139142 ?

Reply:
Yes I noticed this test takes a bit longer on cloud, so I increased the retry.